src/main/java/com/cegeka/academy/service/userChallenge/UserChallengeServiceImpl.java

```diff
@@ -47,4 +47,10 @@ public UserChallenge rateUser(UserChallengeDTO userChallengeDTO, Long ownerId)
             throw new WrongOwnerException();
         }
     }
+
+    @Override
+    public UserChallenge createUserChallenge(UserChallenge userChallenge) {
+
+        return userChallengeRepository.save(userChallenge);
```

Comment:
Asa cum am specificat si mai sus, te rog sa treci in acest serviciu logica de construire a unui UserChallenge